From a4878a2c223b4f49d30d7287b745f9546e2a5697 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 11 Aug 2017 11:50:21 +0300
Subject: [PATCH] Fix "set is already declared in this scope"

../../chrome/browser/devtools/devtools_file_system_indexer.cc:37:12: error: 'set' is already declared in this scope
 using std::set;
---
 chrome/browser/devtools/devtools_file_system_indexer.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/devtools/devtools_file_system_indexer.cc b/chrome/browser/devtools/devtools_file_system_indexer.cc
index 87efb803918d..576a620649dd 100644
--- a/chrome/browser/devtools/devtools_file_system_indexer.cc
+++ b/chrome/browser/devtools/devtools_file_system_indexer.cc
@@ -34,12 +34,13 @@ using base::TimeDelta;
 using base::TimeTicks;
 using content::BrowserThread;
 using std::map;
-using std::set;
 using std::string;
 using std::vector;
 
 namespace {
 
+using std::set;
+
 base::SequencedTaskRunner* impl_task_runner() {
   constexpr base::TaskTraits kBlockingTraits = {base::MayBlock(),
                                                 base::TaskPriority::BACKGROUND};
-- 
2.11.0

